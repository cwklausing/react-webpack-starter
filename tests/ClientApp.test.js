import React from 'react';
import { shallow, configure } from 'enzyme';
import MyComponent from '../scripts/components/ClientApp';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('ClientApp component', () => {
  it('should just render a plain ol snapshot', () => {
    const wrapper = shallow(<MyComponent />);

    expect(wrapper).toMatchSnapshot();
  });
});
