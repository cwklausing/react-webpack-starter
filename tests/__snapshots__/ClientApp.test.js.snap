// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientApp component should just render a plain ol snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyComponent />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Hello world",
    },
    "ref": null,
    "rendered": "Hello world",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Hello world",
      },
      "ref": null,
      "rendered": "Hello world",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
